### PR TITLE
Fix Violations of and Reenable `jsx-a11y/iframe-has-title`

### PR DESCRIPTION
**Pull request by Hamms**
Specifically, by adding titles to all of the identified iframes.

According to [Frames must have title attribute](https://dequeuniversity.com/rules/axe/3.2/frame-title) (linked from the description of the rule itself), it's best practice to have the same text in the title property of the iframe as in the `<title>` element of the embedded document. I attempted to stick to that best practice, and for iframes that are embedding an element with a title, I used that title. Some of our iframes are embedding documents that don't have a title because they are only ever intended to be embedded, never viewed on their own. In those cases, I added a new string describing the nature of the embedded document, and translated it for good measure.

## Links

- https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/iframe-has-title.md

## Testing story

Locally tested some iframes:
- `apps/src/music/views/Video.jsx` at http://localhost-studio.code.org:3000/projectbeats
- `apps/src/p5lab/AnimationTab/PiskelEditor.jsx` at http://localhost-studio.code.org:3000/projects/gamelab/lCyn54D0b0sQ1HyW9zKi2w/edit
- `apps/src/templates/codeDocs/Example.jsx` at http://localhost-studio.code.org:3000/docs/ide/spritelab/expressions/gamelab_setAnimation
- `apps/src/templates/verificationPages/WebLabNetworkCheck.jsx` and `apps/src/weblab/WebLabView.jsx` at http://localhost-studio.code.org:3000/projects/weblab/4c8Iyk7nbYOsFSwdksTzFg/edit

I'm not sure where to test `apps/src/standaloneVideo/Video.tsx`, however; [a comment](https://github.com/code-dot-org/code-dot-org/blob/d365d2afe867ae63ae9966ed1a525aca87824636/apps/src/standaloneVideo/StandaloneVideo.tsx#L3-L6) indicates that we only use this for a subset of levels, and all the levels that I checked are using our [old, jquery-based implementation](https://github.com/code-dot-org/code-dot-org/blob/d365d2afe867ae63ae9966ed1a525aca87824636/apps/src/code-studio/videos.js#L99-L103). Let me know if you know where this is used!

### CODE REVIEW

File: apps/src/music/views/Video.jsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ const Video = ({id, onClose}) => {
             height="100%"
             style={{border: 'none'}}
             src="https://www.youtube-nocookie.com/embed/ab2SBrfkKXU?rel=0"
-            title=""
+            title="Project Beats Introduction"
```
Comment (Hamms) [inline apps/src/music/views/Video.jsx:68]: I considered translating this, but confirmed that the embedded video retains the English title even when viewing youtube in another language. This title is also encoded into the video itself, so sticking to English seems reasonable here.

**Comment by bethanyaconnor** [comment]
Does this level use the `StandaloneVideo` component? http://localhost-studio.code.org:3000/s/allthethings/lessons/46/levels/1

**Comment by Hamms** [comment]
It does, thank you! Tested locally, and confirmed that the title is working.

How did you identify this level, for my own edification?

**Comment by bethanyaconnor** [comment]
It was mostly luck because I'd been looking at the Lab2 `Instructions` component and found out that `Aichat`, `Music`, and a v2 of Dance use Lab2, so I figured a standalone video level in one of the corresponding lessons would use this component. Unfortunately not much magic in that explanation :)

**Review by bethanyaconnor — APPROVED** [review]

**Comment by Hamms** [comment]
Still helpful! I had no idea what Lab2 even was, that narrows it down considerably. Thanks!

**Pull request merged**